dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsCacheImplTest.java

```diff
@@ -0,0 +1,94 @@
+package com.dotcms.security.apps;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.business.DotCacheException;
+import java.util.Arrays;
+import java.util.Map;
+import org.apache.commons.lang.RandomStringUtils;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AppsCacheImplTest {
+
+    static AppsCache cache;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5358201)